indico/modules/rb/schemas.py

```diff
@@ -376,7 +376,7 @@ class CreateBookingSchema(mm.Schema):
     recurrence_weekdays = fields.List(fields.Str(validate=validate.OneOf(WEEKDAYS)))
     room_id = fields.Int(required=True)
     booked_for_user = Principal(data_key='user', allow_external_users=True)
-    booking_reason = fields.String(data_key='reason', validate=validate.Length(min=3), load_default='')
+    booking_reason = fields.String(data_key='reason', validate=validate.Regexp(r'^(?:.{0}|.{3,})$'), load_default='')
```

Comment:
This should not be necessary (and it's super ugly - I'd rather remove the whole validator since it doesn't really matter if someone puts `x` or `xyz` as a dummy booking reason anyway).

Anyway, as long as you make sure that you do not send a value (empty string) for the field (`null` would be fine I think, since that will trigger the `load_default`), this should not be necessary. Something like `reason: reason || null` in `updateBooking` would be  a slightly less ugly way to do it, but I think you can **just add `nullIfEmpty` to the `reason` field in BookingEditForm.jsx when the field is optional** :)

Reply:
I just removed the validation and added `nullIfEmpty `.